src/Chocolatey.Community.Validation.Tests/Rules/IdElementRulesTests.cs

```diff
@@ -7,6 +7,22 @@ namespace Chocolatey.Community.Validation.Tests.Rules
 
     public class IdElementRulesTests : RuleTestBase<IdElementRules>
     {
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase("some-id")]
+        [TestCase("something.portable")]
+        [TestCase("something.commandline")]
+        [TestCase("something.powershell")]
+        [TestCase("something.install")]
+        [TestCase("something.template")]
+        [TestCase("something.extension")]
```

Comment:
Should ".config" also be in this list, since the SubIdentifier method contains it?